src/python/turicreate/test/test_activity_classifier.py

```diff
@@ -150,6 +150,26 @@ def test_create_missing_validation_set(self):
                 validation_set=sf,
             )
 
+    def test_create_single_input_column(self):
+        sf_label = random.randint(0, self.num_labels - 1)
+        sf_session_id = max(self.data[self.session_id])
+        input_data = tc.SFrame(
+                {
+                    self.features[0]: [3.14],
+                    self.target: [sf_label],
+                    self.session_id: [sf_session_id],
+                }
+        )
+        model = tc.activity_classifier.create(
+                input_data,
+                features=["X1-r"],
+                target=self.target,
+                session_id=self.session_id,
+                prediction_window=self.prediction_window,
+        )
+        filename = tempfile.mkstemp("ActivityClassifier.mlmodel")[1]
+        model.export_coreml(filename)
```

Comment:
In addition to making sure we can export the model without error, it would be good to make sure that we can call `predict` (on the exported model) without error and that the results are very close to those from `model.predict` (on the TuriCreate model).

Reply:
Great idea! On it